_monitoring-plugins/ad/index.md

```diff
@@ -140,25 +140,36 @@ If you see any validation errors, edit the settings to fix the errors and then r
 Use the [profile detector]({{site.url}}{{site.baseurl}}/monitoring-plugins/ad/api#profile-detector) operation to make sure you have sufficient data points.
 
 If you see the detector pending in "initialization" for longer than a day, aggregate your existing data using the detector interval to check for any missing data points. If you find a lot of missing data points from the aggregated data, consider increasing the detector interval.
 
-![Anomaly detection results]({{site.url}}{{site.baseurl}}/images/ad.png)
-
 Analyze anomalies with the following visualizations:
 
-- **Live anomalies** - displays live anomaly results for the last 60 intervals. For example, if the interval is 10, it shows results for the last 600 minutes. The chart refreshes every 30 seconds.
-- **Anomaly history** (for historical analysis) / **Anomaly overview** (for real-time results) - plots the anomaly grade with the corresponding measure of confidence.
-- **Anomaly occurrence** - shows the `Start time`, `End time`, `Data confidence`, and `Anomaly grade` for each detected anomaly.
-- **Feature breakdown** - plots the features based on the aggregation method. You can vary the date-time range of the detector.
+- **Live anomalies** (for real-time results) - displays live anomaly results for the last 60 intervals. For example, if the interval is 10, it shows results for the last 600 minutes. The chart refreshes every 30 seconds.
+- **Anomaly overview** (for real-time results) / **Anomaly history**  (for historical analysis in the **Historical analysis** tab) - plots the anomaly grade with the corresponding measure of confidence. This pane includes:
+    - The number of Anomaly occurrences based on the given data-time range. 
+    - The **Average anomaly grade**, a number between 0 and 1 that indicates how anomalous a data point is. An anomaly grade of 0 represents “not an anomaly,” and a non-zero value represents the relative severity of the anomaly. 
+    - **Confidence**, an estimate of the probability that the reported anomaly grade matches the expected anomaly grade. Confidence increases as the model observes more data and learns the data behavior and trends. Note that confidence is distinct from model accuracy.
+    - **Last anomaly occurrence**, the time in which the last anomaly occurred.
+
+Underneath **Anomaly overview** / **Anomaly history** are:
+
+- **Anomaly occurrences** - shows the `Start time`, `End time`, `Data confidence`, and `Anomaly grade` for each detected anomaly.
+- **Feature breakdown** - plots the features based on the aggregation method. You can vary the date-time range of the detector. When selecting a point on the feature line chart, the **Feature output**, the number of times a field appears in your index, and the **Expected value**, a predicted value for the Feature output, appears. Where there is no anomaly, the output and expected values are equal.
```

Comment:
Can you swap the order of `Feature breakdown` section and `Anomaly occurrences` as now the Feature breakdown tab is the default selected tab instead of Anomaly occurrences